### ISSUE TITLE
SA1611 ignores included documentation

### ISSUE THREAD

**Issue by angelobreuer** [DotNetAnalyzers/StyleCopAnalyzers#3173]
The code below produces a violation of SA1611 ([ElementParametersMustBeDocumented](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1611.md)):

`Program.cs`
```csharp
/// <summary>
/// Gets the content of the specified uri as a stream asynchronously.
/// </summary>
/// <param name="uri">the request URI.</param>
/// <returns>
/// A <see cref="Task{TResult}"/> representing the result of the asynchronous operation.
/// </returns>
/// <include file="docs.xml" path="/docs/CancellationToken/*"/>
public Task<Stream> GetContentAsync(Uri uri, CancellationToken cancellationToken = default)
{
     // [...]
}
```

`docs.xml`
```xml
<?xml version="1.0" encoding="utf-8" ?>
<docs>
  <CancellationToken>
    <param name="cancellationToken">
      a cancellation token used to propagate notification that the asynchronous operation
      should be canceled.
    </param>
    <exception cref="System.OperationCanceledException">
      thrown if the cancellation token has had cancellation requested
    </exception>
  </CancellationToken>
</docs>
```

I think the analyzer should read the `docs.xml` file and include the documentation from it but instead, it shows a warning:
`warning SA1611: The documentation for parameter 'cancellationToken' is missing.`

This issue only occurs when some of the parameters are documented in the included and some of the parameters are documented "above the method". IntelliSense recognizes the parameters even if they are documented in different places.

I've also created a repository that shows how to reproduce the issue: https://github.com/angelobreuer/SA1611Repro

Thanks in advance!

**Comment by angelobreuer** [comment]
Grabbing this
